global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "8.0.101"
+    "version": "8.0.200"
```

Comment:
@RussKie are we allowed to change that manually?

Reply:
yes, any time as required